app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,83 @@
+    <RelativeLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="10dp"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <ImageView
+        android:id="@+id/btn_remove"
+        android:layout_width="24dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:layout_alignParentEnd="true"
+        android:contentDescription="@string/remove_description"
+        android:visibility="visible"
+        app:srcCompat="@drawable/ic_clear_24dp" />
+
+      <androidx.appcompat.widget.AppCompatSpinner
         android:id="@+id/spinner_description_languages"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_weight="0"
-        android:minWidth="@dimen/tiny_height"
-        android:padding="@dimen/dimen_0"
+        android:layout_alignParentStart="true"
         android:spinnerMode="dialog" />
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:layout_weight="8">
+
+    </RelativeLayout>
+
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/caption_item_edit_text_input_layout"
+      android:id="@+id/caption_item_edit_text_input_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <androidx.appcompat.widget.AppCompatEditText
+        android:id="@+id/caption_item_edit_text"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        >
-
-        <androidx.appcompat.widget.AppCompatEditText
-            android:id="@+id/caption_item_edit_text"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:hint="@string/share_caption_hint"
-            android:imeOptions="actionNext|flagNoExtractUi"
-            android:inputType="textMultiLine" />
+        android:layout_height="match_parent"
+        android:hint="@string/share_caption_hint"
+        android:imeOptions="actionNext|flagNoExtractUi"
+        android:inputType="textMultiLine" />
     </com.google.android.material.textfield.TextInputLayout>
 
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/description_item_edit_text_input_layout"
+      android:id="@+id/description_item_edit_text_input_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <androidx.appcompat.widget.AppCompatEditText
```

Comment:
We should really replace these with `TextInputEditText`, you aren't supposed to use anything else with TextInputLayout